GetJobber/atlantis
# feat: Announce chip deletion and addition JOB-41252

## Motivations

Deleting a chip is obvious for seeing users. Users that uses screen readers, however, wouldn't know if an item was added or deleted. 

This PR adds a `useLiveAnnounce` hook to announce on screen readers what is happening.

![Jan-28-2022 10-55-32](https://user-images.githubusercontent.com/15986172/151597528-54562356-7e63-472d-8ff7-372781f9f057.gif)


## Changes



### Added

- `useLiveAnnounce` hook

### Changed

- Use said hook to announce when a chip gets added or deleted

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Turn ON mac VoiceOver (or any screen readers really)
- On the Chip dismissible example, add a chip (through shortcut keys or click)
- It should announce that the chip was added
- Delete a chip (through shortcut keys or clicking the x button on the chip)
- It should announce that the chip was deleted

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
